templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-restore-sh.yaml

```diff
@@ -9,26 +9,27 @@ contents:
     set -o pipefail
 
     # example
-    # etcd-snapshot-restore.sh $path-to-snapshot $etcd-connection-string
+    # ./etcd-snapshot-restore.sh $path-to-snapshot $inital_cluster
```

Comment:
spelling?

Reply:
might explain CI failures

Reply:
The param var is misspelled but how would it cause CI failure?

Reply:
oh whoops didn't realize this was commented out my mistake